### PR TITLE
Create sample input username and password then confirm the input

### PR DESCRIPTION
**Pull request by hieupham-thedev**
Create sample input username and password then confirm the input. Please review

### CODE REVIEW

**Comment by MaHai1993** [comment]
add reviewers, assign to yourself

File: src/main/webapp/WEB-INF/views/login.jsp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: MINH HIEU
+  Date: 10/21/2021
+  Time: 2:16 PM
+  To change this template use File | Settings | File Templates.
```
Comment (MaHai1993) [inline src/main/webapp/WEB-INF/views/login.jsp]: Remove comment code

File: src/main/java/com/example/controller/LoginController.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+import java.io.IOException;
+
+@WebServlet(name = "LoginController", value = "/login")
+public class LoginController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```
Comment (MaHai1993) [inline src/main/java/com/example/controller/LoginController.java:24]: add javadocs

**Review by MaHai1993 — COMMENTED** [review]
Fix these comment

**Comment by MaHai1993** [comment]
fix xong push lên lại thì a approve => sẽ merge.

**Comment by hieupham-thedev** [comment]
Done add javadoc and remove comment in jsp file. Please review and approve @MaHai1993

**Review by MaHai1993 — APPROVED** [review]
Looks good to me.

**Comment by MaHai1993** [comment]
already approving, next time remember to resolve conflict

**Pull request merged**